app/plugins/Files/js/loaders.js

```diff
@@ -76,7 +89,9 @@ function uploadDir(dirPath, nickname) {
 					notify('Cannot read ' + file, 'error');
 					return;
 				}
-				if (!isUnixHiddenPath(filePath) && stats.isFile()) {
+				if (file.slice(-4) === '.sia') {
+					loadDotSia(filePath);
```

Comment:
I'm not sure this is a good idea. From the user's perspective, this behavior may be surprising. And what if they actually wanted to upload the .sia file?